src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray`1.cs

```diff
@@ -756,6 +756,11 @@ public ImmutableArray<T> RemoveRange(int index, int length)
             Requires.Range(index >= 0 && index < self.Length, "index");
             Requires.Range(length >= 0 && index + length <= self.Length, "length");
 
+            if (length == 0)
+            {
+                return self;
```

Comment:
What if `this.array == null`? Did this diff change the behavior of RemoveRange to return an uninitialized instance instead of throwing?